CMakeLists.txt

```diff
@@ -210,6 +211,12 @@ if(MINGW)
   add_link_options(-static)
 endif()
 
+if(PCAPPP_BUILD_COVERAGE AND NOT ANDROID)
```

Comment:
Add a comment to explain why the NOT ANDROID

Reply:
Done with [453b322](https://github.com/seladb/PcapPlusPlus/pull/1027/commits/453b322a56b2318a06c21f2909bb923de8204d86)